velox/serializers/RowSerializer.h

```diff
@@ -137,6 +228,47 @@ class RowSerializer : public IterativeVectorSerializer {
 
   memory::MemoryPool* const pool_;
   std::vector<BufferPtr> buffers_;
+
+ private:
+  std::unique_ptr<folly::IOBuf> toIOBuf(const std::vector<BufferPtr>& buffers) {
+    std::unique_ptr<folly::IOBuf> iobuf;
+    for (const auto& buffer : buffers) {
+      auto newBuf =
+          folly::IOBuf::wrapBuffer(buffer->asMutable<char>(), buffer->size());
+      if (iobuf) {
+        iobuf->prev()->appendChain(std::move(newBuf));
+      } else {
+        iobuf = std::move(newBuf);
+      }
+    }
+    return iobuf;
+  }
+
+  int32_t uncompressedSize() const {
```

Comment:
do we want to use a larger data type for this? like int64_t? Thanks!

Reply:
No, because the uncompressedSize in header is int32_t which aligns with PrestoHeader, but I wonder if this is enough. If this is not enough, maybe change both Presto and Row.